src/main/java/org/wildfly/security/auth/spi/AuthenticatedRealmIdentity.java

```diff
@@ -35,6 +36,13 @@
     Principal getPrincipal();
 
     /**
+     * Returns the roles for this identity.
+     *
+     * @return A string set containing the roles for this identity or an empty set if this identity has no roles.
```

Comment:
Completely OT -  @dmlloyd Hi David! Nice set of presentations :) What tool did you use to create them?